### PR TITLE
recovery date-time for deployment 36 and 37 at CE02SHSP

### PR DESCRIPTION
**Pull request by aguirremar**
add full deployment/recovery info for D37 CE02SHSP

### CODE REVIEW

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -360,6 +360,7 @@ AT50-29,Atlantis,2024-08-08T00:00:00,2024-09-09T00:00:00,Cabled (rvdata) Visions
 EK20240814,Elakha,2024-08-14T15:00:00,2024-08-14T23:30:00,Endurance glider 1153-1 recovery and ROV operations
 EK20240815,Elakha,2024-08-15T15:00:00,2024-08-15T18:00:00,Endurance CSPP deployment 
 KH20240821,Kahuna,2024-08-21T10:00:00,2024-08-22T19:30:00,Pioneer 20c glider deployments 
+EK20240830,Elakha,2024-08-30T15:45:00,2024-08-30T19:47:00,Elakha CSPP deployment CE02SHSP-00037
```
Comment (s-pearce) [inline cruise/CruiseInformation.csv:363]: Minor nitpick: the departure time from the cruise quick look report was 15:52 and here you have it as 15:45.
Not important to change, but calling your attention to it, if you meant for both sources to be the same.

File: deployment/CE02SHSP_Deploy.csv [hunk 2]
```diff
@@ -314,12 +314,21 @@ deploymentNumber,Reference Designator,sensor.uid,startDateTime,stopDateTime,lat,
...
-36,CE02SHSP-SP001-07-FLORTJ000,CGINS-FLORTJ-01207,2024-05-02T20:35:00,,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
-36,CE02SHSP-SP001-05-NUTNRJ000,CGINS-NUTNRJ-00870,2024-05-02T20:35:00,,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
-36,CE02SHSP-SP001-04-OPTAAJ000,CGINS-OPTAAJ-00283,2024-05-02T20:35:00,,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
-36,CE02SHSP-SP001-09-PARADJ000,CGINS-PARADJ-00504,2024-05-02T20:35:00,,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
-36,CE02SHSP-SP001-06-SPKIRJ000,CGINS-SPKIRJ-00281,2024-05-02T20:35:00,,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
-36,CE02SHSP-SP001-02-VELPTJ000,CGINS-VELPTJ-12876,2024-05-02T20:35:00,,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
+36,CE02SHSP-SP001-00-SPPENG000,CGCON-CSPPCE-00007,2024-05-02T20:35:00,2024-08-14T19:08:00,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
```
Comment (s-pearce) [inline deployment/CE02SHSP_Deploy.csv:317]: The small boat quick look report says deployment 36 was recovered at 2024-08-14T19:44 and the CSPP worksheet for deployment 36 only has the recovery date in it.  Which is correct?

Comment (s-pearce) [inline deployment/CE02SHSP_Deploy.csv:317]: Cruise EK20240814 is in the cruiseInformation.csv, you could add the recovery cruise ID to the CE02SHSP_Deploy.csv file.

File: deployment/CE02SHSP_Deploy.csv [hunk 3]
```diff
@@ -314,12 +314,21 @@ deploymentNumber,Reference Designator,sensor.uid,startDateTime,stopDateTime,lat,
...
+36,CE02SHSP-SP001-04-OPTAAJ000,CGINS-OPTAAJ-00283,2024-05-02T20:35:00,2024-08-14T19:08:00,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
+36,CE02SHSP-SP001-09-PARADJ000,CGINS-PARADJ-00504,2024-05-02T20:35:00,2024-08-14T19:08:00,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
+36,CE02SHSP-SP001-06-SPKIRJ000,CGINS-SPKIRJ-00281,2024-05-02T20:35:00,2024-08-14T19:08:00,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
+36,CE02SHSP-SP001-02-VELPTJ000,CGINS-VELPTJ-12876,2024-05-02T20:35:00,2024-08-14T19:08:00,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
+37,CE02SHSP-SP001-00-SPPENG000,CGCON-CSPPCE-00002,2024-08-30T17:57:00,2024-09-18T16:25:00,44.63611,-124.30050,CGMCE-02SHSP-00037,EK20240830,CGVEH-CSPPCE-00002,1,,,,,83,83,,,
+37,CE02SHSP-SP001-08-CTDPFJ000,CGINS-CTDPFJ-00336,2024-08-30T17:57:00,2024-09-18T16:25:00,44.63611,-124.30050,CGMCE-02SHSP-00037,EK20240830,CGVEH-CSPPCE-00002,1,,,,,83,83,,,
```
Comment (s-pearce) [inline deployment/CE02SHSP_Deploy.csv:327]: @cwingard Are there not CTDPF-J calibrations in Asset Management?  There isn't even a CTDPFJ directory.

File: deployment/CE02SHSP_Deploy.csv [hunk 4]
```diff
@@ -314,12 +314,21 @@ deploymentNumber,Reference Designator,sensor.uid,startDateTime,stopDateTime,lat,
...
+37,CE02SHSP-SP001-01-DOSTAJ000,CGINS-DOSTAJ-00352,2024-08-30T17:57:00,2024-09-18T16:25:00,44.63611,-124.30050,CGMCE-02SHSP-00037,EK20240830,CGVEH-CSPPCE-00002,1,,,,,83,83,,,
+37,CE02SHSP-SP001-07-FLORTJ000,CGINS-FLORTJ-01519,2024-08-30T17:57:00,2024-09-18T16:25:00,44.63611,-124.30050,CGMCE-02SHSP-00037,EK20240830,CGVEH-CSPPCE-00002,1,,,,,83,83,,,
+37,CE02SHSP-SP001-05-NUTNRJ000,CGINS-NUTNRJ-00425,2024-08-30T17:57:00,2024-09-18T16:25:00,44.63611,-124.30050,CGMCE-02SHSP-00037,EK20240830,CGVEH-CSPPCE-00002,1,,,,,83,83,,,
+37,CE02SHSP-SP001-04-OPTAAJ000,CGINS-OPTAAJ-00138,2024-08-30T17:57:00,2024-09-18T16:25:00,44.63611,-124.30050,CGMCE-02SHSP-00037,EK20240830,CGVEH-CSPPCE-00002,1,,,,,83,83,,,
```
Comment (s-pearce) [inline deployment/CE02SHSP_Deploy.csv:331]: The last OPTAA-J calibration I see for SN 0138 in AM is 2021.  Are there more recent calibrations to enter?

**Review by s-pearce — COMMENTED** [review]
@aguirremar, there are a few questions I have in the review before approving. Can you respond to these either in person or here in Github, and I'll approve otherwise.

File: deployment/CE02SHSP_Deploy.csv (hunk 3 above)
Comment (cwingard) [reply deployment/CE02SHSP_Deploy.csv:327]: @s-pearce as we have discussed before, no the CTDPF does not have calibration coefficients that go into AM. At this point, if there's no folder for that instrument class, its a pretty safe bet you can ignore it.

File: deployment/CE02SHSP_Deploy.csv (hunk 2 above)
Comment (cwingard) [reply deployment/CE02SHSP_Deploy.csv:317]: This is not something that has been done for any of the CSPPs. Indeed, it wasn't even been done for the moorings until I started doing it last year. Let's put this one on hold and come up with a plan to get all CSPP deployments updated with the recovery cruises, and then start adding them going forward as a SOP.

**Comment by cwingard** [comment]
@aguirremar let's get the three remaining issues noted by Stuart addressed and then we can merge. Highest priority is tracking down the missing cal (if indeed missing) for AC-S 138.

File: deployment/CE02SHSP_Deploy.csv (hunk 2 above)
Comment (s-pearce) [reply deployment/CE02SHSP_Deploy.csv:317]: Agreed. Just trying to start a good habit and the cruise ID was already in AM.

File: deployment/CE02SHSP_Deploy.csv (hunk 3 above)
Comment (s-pearce) [reply deployment/CE02SHSP_Deploy.csv:327]: I forgot I asked before.  By focusing on it, I'm trying to get the wheels in motion for adding it in eventually.

Comment (cwingard) [reply deployment/CE02SHSP_Deploy.csv:327]: No, we will not be adding it.

File: cruise/CruiseInformation.csv (hunk 1 above)
Comment (aguirremar) [reply cruise/CruiseInformation.csv:363]: changed all places to 15:52 to match quicklook report

File: deployment/CE02SHSP_Deploy.csv (hunk 2 above)
Comment (aguirremar) [reply deployment/CE02SHSP_Deploy.csv:317]: 19:44 is correct, it just wasn't showing on the Excel spreadsheet, it is corrected now.

File: deployment/CE02SHSP_Deploy.csv (hunk 4 above)
Comment (cwingard) [reply deployment/CE02SHSP_Deploy.csv:331]: Indeed, last cal

**Review by cwingard — APPROVED** [review]

**Review by s-pearce — APPROVED** [review]
Looks good.

**Pull request merged**